README.md

```diff
@@ -409,6 +437,25 @@ Supports importing of external functions and data using bindings.
     
     - In the above example: **processECommerace** step is the workflow step and importing additional bindings. Both workflow’s (**commonBinding**) and step’s (**stepBinding**) bindings are available to the workflow step.
 
+### Batch Step
+This helps to batch the inputs by length or size. We can also define our own batching by implementing [**BatchExecutor**](./src/steps/types.ts#L114) interface.
+
```

Comment:
Based on filter the key also will be returned with the batched output we can mention this as well